assets/app/scripts/controllers/route.js

```diff
@@ -62,6 +63,11 @@ angular.module('openshiftConsole')
             };
           });
 
+        // List services to display route warnings.
+        DataService.list("services", context, function(services) {
```

Comment:
so if the service is updated and the warning is no longer valid then we wouldnt know to update the warning?  its just inconsistent with other pages where warnings can go away while you are on the page